lib/ansible/modules/package.py

```diff
@@ -40,6 +40,8 @@
     description:
       - The required package manager module to use (V(dnf), V(apt), and so on). The default V(auto) will use existing facts or try to autodetect it.
       - You should only use this field if the automatic selection is not working for some reason.
+      - Since version 2.17 you can override the automatic detection both via a local facts by setting 'pkg_mgr' or override both by using the
```

Comment:
```suggestion
      - Since version 2.17 you can override the automatic detection both through a local facts by setting ``pkg_mgr`` or override both by using the
```

Reply:
@samccann as a bilingual spanish speaker that is not proficient in either language, its really hard to me sometimes to even realize it is a latinism